### DISCUSSION TITLE
Without i18n, let me set the lang attribute of the html tag

### DISCUSSION THREAD

**Discussion by sebsonjura** [vercel/next.js#36504] [Ideas]
### Describe the feature you'd like to request

When producing a site as a static export with next.js that only supports one language, please still let me set the language attribute of the page, without diving into the depths of i18n.

### Describe the solution you'd like

a simple config option. 

### Describe alternatives you've considered

* using a post build file that changes all html base tags (slow build at 7863 pages) 
* checked out i18n, but we do not add language information to our routes

**Comment by ijjk** [comment]
Hi, this is possible without `i18n` config by adding a `lang="en"` attribute on the `Html` element in a custom document, related documentation: https://nextjs.org/docs/advanced-features/custom-document

Note: if you are only generating one language the i18n config needed for that is:
```js
module.exports = {
  i18n: {
    locales: ['en'],
    defaultLocale: 'en'
  }
}
```

**Reply by ladisays** [reply]
Hi @ijjk, we already tried adding that to the `<Html>` component inside the `_document.js` but as you already know, the custom document does not support `getStaticProps`. And we rely heavily on `getStaticProps` for static export. 

We're currently having to do a string replacement after running `next export` in order to be able to add the `lang` attribute to the `html` tag, which is not efficient.

We also have some static pages that do not make use of `getStaticProps`.
